pyaedt/modeler/cad/Primitives.py

```diff
@@ -275,45 +587,4908 @@ def version(self):
+                                        if isinstance(op, (OrderedDict, dict)):
+                                            if op["ID"] == op_id:
+                                                props = op["FaceCSParameters"]
+                                                coord.append(FaceCoordinateSystem(self, props, name))
+                                        elif isinstance(op, list):
+                                            for iop in op:
+                                                if iop["ID"] == op_id:
+                                                    props = iop["FaceCSParameters"]
+                                                    coord.append(FaceCoordinateSystem(self, props, name))
+                                                    break
+                except:
+                    pass
+            for cs in coord:
+                if isinstance(cs, CoordinateSystem):
+                    try:
+                        cs._ref_cs = id2name[name2refid[cs.name]]
+                    except:
+                        pass
+        coord.reverse()
+        return coord
+
+    def _get_lists_data(self):
+        """Retrieve user object list data.
+
+        Returns
+        -------
+        [Dict with List information]
+        """
+        design_lists = []
+        if self._design_properties and self._design_properties.get("ModelSetup", None):
+            key1 = "GeometryOperations"
+            key2 = "GeometryEntityLists"
+            key3 = "GeometryEntityListOperation"
+            try:
+                entity_list = self._design_properties["ModelSetup"]["GeometryCore"][key1][key2]
+                if entity_list:
+                    geom_entry = copy.deepcopy(entity_list[key3])
+                    if isinstance(geom_entry, (dict, OrderedDict)):
+                        geom_entry = [geom_entry]
+                    for data in geom_entry:
+                        props = {}
+                        name = data["Attributes"]["Name"]
+                        props["ID"] = data["ID"]
+                        props["Type"] = data["GeometryEntityListParameters"]["EntityType"]
+                        if props["Type"] == "Object":
+                            name_list = []
+                            for element in data["GeometryEntityListParameters"]["EntityList"]:
+                                element_name = self.oeditor.GetObjectNameByID(int(element))
+                                name_list.append(element_name)
+                            props["List"] = name_list
+                        else:
+                            props["List"] = data["GeometryEntityListParameters"]["EntityList"]
+                        design_lists.append(Lists(self, props, name))
+            except:
+                self.logger.info("Lists were not retrieved from AEDT file")
+        return design_lists
+
+    def _get_planes_data(self):
+        """Retrieve planes data.
```

Comment:
```suggestion
        """Get the plane's data.
```
I assume it is the data for one plane, so I changed it to an apostrophe and ''s".